pagopa/io-app
# feat(IDPay): [IODPAY-43] Add initiatives cards in wallet screen

## Short description
Added the ability to see onboarded initiatives in the wallet section of the app. In this PR there are still some temporary UI assets.
<img src="https://user-images.githubusercontent.com/6160324/202203771-75e2a3ac-1569-4c86-aba8-07f8adf31da3.png" width=200 />



## List of changes proposed in this pull request
- Added reducer, actions and sagas to handle remote data requests
- Added `IDPayCardsInWalletContainer` component
- Added `IDPayCardPreviewComponent` component
- Added locales
- Added tests

## How to test
- Enable the IDPAY_ENABLED feature flag
- Complete an initiative onboarding from the IDPay onboarding playground in the profile section
- Check the wallet section
- Run `yarn jest` to test the reducer and sagas
